pandas/src/algos_groupby_helper.pxi.in

```diff
@@ -568,6 +568,76 @@ def group_min_{{name}}(ndarray[{{dest_type2}}, ndim=2] out,
                 else:
                     out[i, j] = minx[i, j]
 
+
+@cython.boundscheck(False)
+@cython.wraparound(False)
+def group_cummin_{{name}}(ndarray[{{dest_type2}}, ndim=2] out,
+                          ndarray[{{dest_type2}}, ndim=2] values,
+                          ndarray[int64_t] labels,
+                          ndarray[{{dest_type2}}, ndim=2] accum):
```

Comment:
so I am puzzled why we even pass in ``accum`` here at all. It is *only* used internally. This would affect the groupby code and cumsum, cumprod as well as these new funcs.

@mroeschke if you want to do a followup to remove it after would be great.